src/hotspot/cpu/aarch64/aarch64_sve.ad

```diff
@@ -3061,3 +3059,37 @@ instruct extractL(iRegLNoSp dst, vReg src, immI idx,  pRegGov pTmp, rFlagsReg cr
   %}
   ins_pipe(pipe_slow);
 %}
+
+instruct extractF(vRegF dst, vReg src, immI idx,  pRegGov pTmp, rFlagsReg cr)
```

Comment:
Remove the redundant space between `idx,  pRegGov`?

Reply:
Sure, thank you for your review.